tests/release/e2e-test-default-bundle.go

```diff
@@ -0,0 +1,240 @@
+		Expect(err).NotTo(HaveOccurred(), "Error when creating devNamespace: %v", err)
+		klog.Info("Dev Namespace created: ", devNamespace)
+
+		_, err = framework.CommonController.CreateTestNamespace(managedNamespace)
+		Expect(err).NotTo(HaveOccurred(), "Error when creating managedNamespace: %v", err)
+		klog.Info("Managed Namespace created: ", managedNamespace)
+
+		klog.Infof("Wait until the 'pipeline' SA is created in %s namespace \n", managedNamespace)
+		Eventually(func() bool {
+			sa, err := framework.CommonController.GetServiceAccount("pipeline", managedNamespace)
+			return sa != nil && err == nil
+		}, oneMinuteTimeout, defaultInterval).Should(BeTrue(), "timed out when waiting for the \"pipeline\" SA to be created")
+
+		sourceAuthJson := utils.GetEnv("QUAY_TOKEN", "")
+		Expect(sourceAuthJson).ToNot(BeEmpty())
+
+		destinationAuthJson := utils.GetEnv(constants.QUAY_OAUTH_TOKEN_RELEASE_DESTINATION, "")
+		Expect(destinationAuthJson).ToNot(BeEmpty())
+
+		_, err = framework.CommonController.CreateRegistryAuthSecret(hacbsReleaseTestsTokenSecret, devNamespace, sourceAuthJson)
+		Expect(err).ToNot(HaveOccurred())
+
+		_, err = framework.ReleaseController.CreateRegistryJsonSecret(redhatAppstudioUserSecret, managedNamespace, destinationAuthJson, destinationKeyName)
+		Expect(err).ToNot(HaveOccurred())
+
+		err = framework.CommonController.LinkSecretToServiceAccount(devNamespace, hacbsReleaseTestsTokenSecret, "pipeline")
+		Expect(err).ToNot(HaveOccurred())
+
+		klog.Info("Dev Namespace:", devNamespace)
+		klog.Info("Managed Namespace:", managedNamespace)
+
+		// Copy the public key from tekton-chains/signing-secrets to a new
+		// secret that contains just the public key to ensure that access
+		// to password and private key are not needed.
+		publicKey, err := kubeController.GetPublicKey("signing-secrets", constants.TEKTON_CHAINS_NS)
+		Expect(err).ToNot(HaveOccurred())
+		Expect(kubeController.CreateOrUpdateSigningSecret(
+			publicKey, publicSecretNameAuth, managedNamespace)).To(Succeed())
+
+		defaultEcPolicy = ecp.EnterpriseContractPolicySpec{
+			Description: "Red Hat's enterprise requirements",
+			PublicKey:   string(publicKey),
+			Sources: []ecp.Source{
+				{
+					Name: "ec-policies",
+					Policy: []string{
+						"git::https://github.com/hacbs-contract/ec-policies.git//policy",
+					},
+					Data: []string{
+						"git::https://github.com/hacbs-contract/ec-policies.git//data",
+					},
+				},
+			},
+			Exceptions: &ecp.EnterpriseContractPolicyExceptions{
+				NonBlocking: []string{"tasks", "attestation_task_bundle", "java", "test", "not_useful"},
+			},
+		}
+
+	})
+
+	AfterAll(func() {
+
+		if !CurrentSpecReport().Failed() {
+			Expect(framework.CommonController.DeleteNamespace(devNamespace)).NotTo(HaveOccurred())
+			Expect(framework.CommonController.DeleteNamespace(managedNamespace)).NotTo(HaveOccurred())
+		}
+	})
+
+	var _ = Describe("creating all CRDs required for test", func() {
```

Comment:
This looks like what we should test is the `Creating and configuring the managed namespace`

Reply:
based on your suggestion, I've moved all creation and configuring the CRDs to the BeforeAll, 
it was tested locally and it's working fine